src/Admin/OnboardingWizardSubmenu.php

```diff
@@ -40,7 +40,7 @@ public static function get_registration_action() {
 	 */
 	public function register() {
 		add_submenu_page(
-			'',
+			'options.php',
```

Comment:
Linking your pervious https://github.com/ampproject/amp-wp/pull/7614#discussion_r1323144193 here.

This was made an empty string in https://github.com/ampproject/amp-wp/commit/e30168c9a2aed5adfdaa0c3d018cf10f520e2008.

You say:

> But adding `options.php` provides the same behavior without breaking anything. So updating it to `options.php`.

Can you clarify further what this is doing? Is it that there is no menu item for `options.php` so it won't add anything to the menu?

Reply:
Yes. It won't be accessible from a  submenu item but rather only accessible via the URL which is `options.php?page=amp-onboarding-wizard`.